pom.xml

```diff
@@ -22,7 +22,7 @@
   <properties>
     <revision>1.3</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.73.3</jenkins.version>
```

Comment:
Yikes! Without changelog or pull request description saything why this change is needed I cannot approve.

Reply:
I added the explanation in the PR description. I hope it helps.

Reply:
change seems unrelated to the proposed contribution, please void mixing content.
also, I don't get the point: we already run integration test of jobdsl plugin without this core version bump being needed.